### DISCUSSION TITLE
PRACTICA - HTML - PASTELERIA

### DISCUSSION THREAD

**Discussion by jlnunezobando** [Launch-X-Latam/MisionFrontEnd#395] [Show and tell]
Buenas a todos mi compañeros de Launch X, estoy realizando todas practicas y apenas acabe la de la pastelería, claramente me falta mucho para ser estéticamente correcta jajaja,

Agradezco la experiencia de los compañeros para mejorar esta pagina, use la guía de la clase de cherpar, muchas gracias a todos y no muriendo en el intento.

Repositorio porque no se como funciona la pagina :D
https://github.com/jlnunezobando/1html-pasteleria.git

**Comment by RIvanCF** [comment]
Sugiero que veas unos tutoriales para hacer uso de GitHub Pages y así podamos visualizar tu proyecto, ya que si solo pasas el puro repositorio es como si visualizáramos el código, pero no tus resultados
